src/model_config_utils.cc

```diff
@@ -784,6 +784,27 @@ GetNormalizedModelConfig(
   return Status::Success;
 }
 
+Status
+SetDefaultInstanceCount(
+    inference::ModelInstanceGroup* group, const std::string& backend)
+{
+  group->set_count(1);
+
+  // Backends opt into the default_cpu_instance_count since
+  // some backends (pytorch, OpenVINO) don't perform well/have high overhead
+  // when using multiple instances.
+  const int default_cpu_instance_count = 2;
+  bool use_default_cpu_instance_count = (backend == kTensorRTBackend) ||
```

Comment:
TensorRT backend runs only on GPU and can not run on CPU.
So, KIND can not be KIND_CPU for TensorRT.
It is fine removing that from these conditions.

Reply:
Updated to take out TensorRT from the opt in list.